[ci-visibility] Add instructions for cypress>=10 (#14221)
DataDog/documentation: content/en/continuous_integration/setup_tests/javascript.md

{{% tab "Cypress" %}}

### Cypress<10

> These are the instructions if you're using a version lower than `cypress@10`.

Reviewer: ```suggestion
These are the instructions if you're using a version older than `cypress@10`.
```

DD_ENV=ci DD_SERVICE=my-ui-app npm test
{{< /code-block >}}

### Cypress >=10

> The API for `cypress>=10` can be seen at the [cypress docs][4].

Reviewer: ```suggestion
Use the Cypress API documentation to [learn how to write plugins][4] for `cypress>=10`.
```

> In your `cypress.config.js` file:

Reviewer: ```suggestion
In your `cypress.config.js` file, set the following:
```

{{< code-block lang="javascript" filename="cypress.config.js" >}}
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  e2e: {
    setupNodeEvents: require('dd-trace/ci/cypress/plugin'),
    supportFile: 'cypress/support/index.js'
  }
})
{{< /code-block >}}

Your `supportFile` should look the same as in `cypress<10`:

{{< code-block lang="javascript" filename="cypress/support/index.js" >}}
// your previous code is before this line
require('dd-trace/ci/cypress/support')
{{< /code-block >}}

> If you're already using other cypress plugins:

Reviewer: ```suggestion
If you're using other cypress plugins, your `cypress.config.js` file should contain the following:
```
